collect_app/src/androidTest/java/org/odk/collect/android/support/pages/Page.java

```diff
@@ -78,12 +77,6 @@
 
 abstract class Page<T extends Page<T>> {
 
-    final ActivityTestRule rule;
-
-    Page(ActivityTestRule rule) {
```

Comment:
This is the change that is responsible for loads of other files changing. We'd previously needed `rule` to deal with getting an Activity for string translation but weren't using it anymore. This means that our pages can work without an `ActivityTestRule` and means that we can now use `AcitivityScenario` instead.